Minio.Examples/Program.cs

```diff
@@ -17,85 +17,165 @@
 
-                //Cases.GetBucketPolicy.Run(minioClient, bucketName).Wait();
+                // Delete the file and Download the object as file
+                Cases.GetObject.Run(minioClient, bucketName, objectName, smallFileName).Wait();
+               
+                // Delete the file and Download partial object as file
+                Cases.GetPartialObject.Run(minioClient, bucketName, objectName, smallFileName).Wait();
 
-                //Cases.SetBucketPolicy.Run(minioClient, bucketName).Wait();
-                //Cases.StatObject.Run(minioClient, bucketName, objectName).Wait();
-                //Cases.CopyObject.Run(minioClient, bucketName, objectName, destBucketName, destObjectName).Wait();
+                // Server side copyObject
+                Cases.CopyObject.Run(minioClient, bucketName, objectName, destBucketName, objectName).Wait();
+
+                // Upload a File with PutObject
+                Cases.FPutObject.Run(minioClient, bucketName, objectName, smallFileName).Wait();
+
+                // Delete the file and Download the object as file
+                Cases.FGetObject.Run(minioClient, bucketName, objectName, smallFileName).Wait();
+
+                // Automatic Multipart Upload with object more than 5Mb
+                Cases.PutObject.Run(minioClient, bucketName, objectName, bigFileName).Wait();
+
+                // List the incomplete uploads
+                Cases.ListIncompleteUploads.Run(minioClient, bucketName);
+
+                // Remove all the incomplete uploads
+                Cases.RemoveIncompleteUpload.Run(minioClient, bucketName, objectName).Wait();
+
+                // Set a policy for given bucket
+                Cases.SetBucketPolicy.Run(minioClient, PolicyType.READ_ONLY, bucketName).Wait();
+
+                // Get the policy for given bucket
+                Cases.GetBucketPolicy.Run(minioClient, bucketName).Wait();
+
+                // Get the presigned url for a GET object request
+                Cases.PresignedGetObject.Run(minioClient, bucketName, objectName).Wait();
+
+                // Get the presigned POST policy curl url
+                Cases.PresignedPostPolicy.Run(minioClient).Wait();
+
+                // Get the presigned url for a PUT object request
+                Cases.PresignedPutObject.Run(minioClient, bucketName, objectName).Wait();
+
+
+                // Delete the object
+                Cases.RemoveObject.Run(minioClient, bucketName, objectName).Wait();
+
+                // Delete the object
+                Cases.RemoveObject.Run(minioClient, destBucketName, objectName).Wait();
+
+                // Remove the buckets
+                Cases.RemoveBucket.Run(minioClient, bucketName).Wait();
+                Cases.RemoveBucket.Run(minioClient, destBucketName).Wait();
+
+                // Remove the binary files created for test
+                File.Delete(smallFileName);
+                File.Delete(bigFileName);
```

Comment:
In the other SDKs, examples are being written as separate self-contained programs - here it is one big program that runs all the examples. IMO it is simpler for library users to read/use examples from self-contained programs.